conf/application.conf

```diff
@@ -170,6 +170,20 @@ microservice {
       refreshAfter = 10
       expireAfter = 60
     }
+
+    tai {
```

Comment:
There is already a tai section in the config and the port numbers are different. I also can't see where bearer-token in used. Might be better to just include the circuit breaker specific config to the current tai config and  remove the the rest from here